README.md

```diff
@@ -3,336 +3,91 @@
 ![ci](https://github.com/valorem-labs-inc/valorem-core/actions/workflows/CI.yml/badge.svg)
 [![codecov](https://codecov.io/gh/valorem-labs-inc/valorem-core/branch/master/graph/badge.svg?token=M52NC4Q3SW)](https://codecov.io/gh/valorem-labs-inc/valorem-core)
 
-This repository contains a binary smart contract system comprised of many libraries,
-which together make the Valorem Options V1 Core. The Core contracts provide an option
-settlement engine upon which more complex systems can be built.
-
-The Core is designed to be gas efficient, minimal, and provide a secure settlement
-system. The Core consists, primarily, of a settlement engine which allows users
-to write options, exercise options, redeem claims for assets, and settles assignments
-of exercises to claims written.
+[Valorem](https://valorem.xyz/) is a DeFi money lego, enabling writing covered call and covered put, physically settled, American, European, or Exotic options.
+
+### Table of Contents
+
+- [Introduction](#introduction-to-the-protocol)
+- [Trust Model](#trust-model)
+- [Cucumber Features](./test/features)
+- [Building the Project](#building-the-project)
+- [Security Contact Info](#security-contact-info)
+
+## Introduction to the Protocol
+
+The Valorem Options V1 Core consists of a settlement engine which allows users to write options, exercise options, and redeem claims on assets, while handling fair assignment of exercises to claims written. It is designed to be gas efficient, minimal, and provide a secure settlement layer upon which more complex systems can be built.
+
+The Settlement Engine follows the [ERC-1155 multi-token](https://eips.ethereum.org/EIPS/eip-1155) standard. Options can be written for any pair of valid ERC-20 assets (excluding rebasing, fee-on-transfer, and ERC-777 tokens). When written, an options contract is represented by semi-fungible Option tokens, which can be bought/sold/transferred between addresses like any ERC-1155 token.
+
+An option writer's claim to the underlying asset(s) (if not exercised) and exercise asset(s) (if exercised) is represented by a non-fungible Option Lot Claim token. This Claim NFT can be redeemed for their share of the underlying plus exercise assets, based on currently exercised.
```

Comment:
Can we just say `option lot Claim token`?

Reply:
So lower caps on option lot? Done